README.md

```diff
@@ -44,26 +44,20 @@ node            |  |                               |  |
 ```sh
 # Print help message
 celestia-appd --help
-```
-
-### Create your own single node devnet
-
-```sh
-# WARNING: this deletes config, data, and keyrings from previous local devnets
-rm -r ~/.celestia-app
 
-# Start a single node devnet
-./scripts/single-node.sh
+# Create your own single node devnet
+celestia-appd init test --chain-id test
+celestia-appd keys add user1
+celestia-appd add-genesis-account <address from above command> 10000000utia,1000token
+celestia-appd gentx user1 1000000utia --chain-id test
+celestia-appd collect-gentxs
+celestia-appd start
```

Comment:
we actually want to keep the changes to the README

Reply:
Then let's have #977 as it already contains necessary changes